cmd/influx/importer/v8.go

```diff
@@ -0,0 +1,222 @@
+package importer
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/influxdb/influxdb/client"
+)
+
+const batchSize = 5000
+
+// V8Config is the config used to initialize a V8 importer
+type V8Config struct {
+	username, password string
+	url                url.URL
+	precision          string
+	writeConsistency   string
+	file, version      string
+	compressed         bool
+}
```

Comment:
Why are these unexported? It seems like it'd be easier to declare exported so that you can set them individually instead of passing through a constructor.